src/change_list/mod.rs

```diff
@@ -235,18 +235,18 @@ impl ChangeListBuilder<'_> {
 
     pub fn set_attribute(&mut self, name: &str, value: &str) {
         debug_assert!(self.traversal_is_committed());
```

Comment:
Here's where the SVG change begins and ends. I believe it's just this one if statement.